.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:   
+      interval: "weekly"  
```

Comment:
You missed a newline at EOF.

Reply:
Will fix! Dependabot config didn't fail, so I thought it wasn't an issue.

Reply:
It's not an issue, just a code quality thing.

Reply:
Alright.